Feature/add html header headline

Shahid-Bagwan opened the pull request:
## Pull Request: Professionalizing Changes in HTML & CSS Files

### Changes Made
- Added a header section to the HTML file.
- Added a headline section to the HTML file.
- Designed the header and headline sections in the CSS file.
- Updated the README file with relevant project information.

### Description
This pull request aims to professionalize the changes made in the HTML and CSS files. The following modifications have been implemented:

#### HTML Changes
1. Added a header section to provide consistent navigation across the website.
2. Included a headline section to grab user attention and provide a brief introduction.

#### CSS Changes
1. Designed the header section to enhance the visual appeal and improve user experience.
2. Styled the headline section to ensure readability and a coherent design with the rest of the webpage.

#### README Update
Added information related to the project in the README file. This includes relevant details about the purpose, features, and usage of the project. The updated README aims to provide comprehensive documentation for developers and users.

### Additional Information
Any relevant background context or information regarding the changes made can be included here.

Please review this pull request and provide any feedback or suggestions for improvement.

Shahid-Bagwan committed:
update index, style & readme files.

created the header & headline section, added images for designing in resource folder. also added documentation of the project in the readme file

Shahid-Bagwan committed:
Uploading readme

uploaded Readme file because it did not get pushed in the previous commit

Shahid-Bagwan committed:
update readme

removed an a unwanted file.

Shahid-Bagwan committed:
Fix : Webhint Error

Removed Inline css and added the css in style.css

jmonto55 commented on index.html:
```diff
@@ -1,13 +1,49 @@
+            <img class="img1" src="resources/Logo.png" alt="Logo Icon">
+            <img class="img2" src="resources/menu.png" alt="Menu Icon">
+        </nav>
+    </header>
```
- [ ] Great work building your `<header>` 💯. However, please kindly ensure to extract your logo from the `<nav>`  tag and leave only the burger menu inside of it. 🤓

jmonto55 commented on style.css:
```diff
@@ -1,4 +1,96 @@
-  padding: 0;
+}
+
+header {
+  background-color: #3c3c3c;
+}
```
- [ ] Good work styling your header ✔️. However, please kindly ensure to apply the correct margin to stick as much as possible to the Figa design. 🤓

Figma design 👇🏽
![f1](https://github.com/Shahid-Bagwan/Portfolio/assets/116701485/c0a1414e-bc8f-437c-a382-38f0ffe4fc68)

Your deign 👇🏽
![d1](https://github.com/Shahid-Bagwan/Portfolio/assets/116701485/8d1fb1a5-c381-4051-aa33-b41e49e58766)

jmonto55 commented on index.html:
```diff
@@ -1,13 +1,49 @@
+                    <li class="lists"><i class="fa-brands fa-facebook" ></i></li>
+                    <li class="lists"><i class="fa-brands fa-twitter" ></i>></li>
+                    <li class="lists"><i class="fa-brands fa-instagram" ></i></li>
+                </ul>
```
- [ ] Excellent job using `<ul>` and `<li>` semantic tags to build your list 🚀. However, please kindly ensure to use `<a>` tags instead of `<i>` tags to wrap your social media tags. 🤓 In the near Milestones you will be adding your social media link to these icons, using the anchor tag will be helpful for this. 🔧
> Hint 👇🏽
```
<ul>
  <li><a href="#">First item</a></li>
  <li><a href="#">Second item</a></li>
  <li><a href="#">Third item</a></li>
</ul>
```

jmonto55 commented on style.css:
```diff
@@ -1,4 +1,96 @@
+    display: flex;
+    width: 200px;
+    justify-content: space-around;
+  }
```
- [ ] Great work applying styles to your social media list. 💯 However, please kindly ensure to:
- Use the same icons provided in the Figma design. 🤓
- Use the same margin-left following the Figma design. 🔧

jmonto55 commented on style.css:
```diff
@@ -1,4 +1,96 @@
+    color: white;
+    margin: 0 0 10px 16px;
+    width: 375px;
+  }
```
- [ ] Good work styling your paragraph here ✔️. However, I noticed it is overlapping with the background image. 🤓

Figma design 👇🏽
![f](https://github.com/Shahid-Bagwan/Portfolio/assets/116701485/acb0619c-f7f0-496d-bc14-1b06b355b7ab)

Your design 👇🏽
![d](https://github.com/Shahid-Bagwan/Portfolio/assets/116701485/7efccf45-3d7e-4934-8e5d-54fad308c79e)

jmonto55 reviewed (changes requested):
# STATUS: REQUIRED CHANGES 🟡

Hi  @Shahid-Bagwan 👋🏽, 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!

![AlmostThereGIF (2)](https://user-images.githubusercontent.com/116701485/232347575-8b3b93b1-a382-4b9e-82bd-30892aad7d39.gif)




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Shahid-Bagwan committed:
Update Index & Style file.

Made the following Changes Suggested by the code reviewer :

- remove the logo image from <nav> tag.

- add margin to logo and hamburger img.

- replace <i> tag with <a>

- remove personal icons & add icons from the figma desing.

- remove image which was overlapping.

Shahid-Bagwan commented:
**@jmonto55 made suggested changes. is there anything left Or I have missed out on?**

jmonto55 commented:
> **@jmonto55 made suggested changes. is there anything left Or I have missed out on?**

Hi @Shahid-Bagwan , it is great that you managed to implement the changes, please submit your project again to receive a second review. 🎯

victorokoroji reviewed (changes requested):
Hi @Shahid-Bagwan, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️

- [x] Kindly implement all the required changes suggested by the previous reviewer. If you have done so, kindly push the changes to Git Hub. 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Shahid-Bagwan committed:
Merge pull request #3 from Shahid-Bagwan/features

Update Index & Style file.

Shahid-Bagwan commented:
@vickymarz done!

franclobo commented on style.css:
```diff
@@ -1,4 +1,100 @@
+    font-family: 'Poppins', sans-serif;
+    word-wrap: break-word;
+    color: white;
+    margin: 0 0 0 16px;
+    width: 375px;
```
- [ ] I suggest correcting the margin and removing the width of the paragraph to be similar to the Figma design 👇🏽 

```css
.paragraph {
  margin: 0 16px;
}
```

| Figma design | Your design |
|---|---|
| ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/60d7ca37-c42c-4d6a-a4f5-4ac08316675a) | ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/750bb8e7-39ff-4169-a2dc-753249797fa6) |

Shahid-Bagwan replied:
@franclobo Done! should I merge it now!

franclobo reviewed (changes requested):
Hi @Shahid-Bagwan, 

# ♻️ Required Changes ♻️

![so-close-jim-halpert](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/08778f81-72a3-4890-92f0-8d1df1e66a90)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Good practices of HTML & CSS ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Shahid-Bagwan committed:
Update Style.css

Made changes suggested by the code reviewer

shella12 commented on index.html:
```diff
@@ -1,13 +1,47 @@
+                    <li class="lists"><a href="#"  > <img src="./resources/logo5.png" alt="logo"> </a></li>
+                </ul>
+            </div>
+            <div class="image_container">
+                <img src="resources/Illustration.png" alt="image">
+            </div>
```
- [ ] [OPTIONAL]  I would suggest that if an image is not part of the content (aesthetic reasons) use CSS `background-image` instead of `img` tag (decorations are always background images, not img tags). [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

- [ ] [OPTIONAL]  I would suggest you to remove `div` tag from around  `img` tag  .  Unless you need to include the `div` for some structural/organisational reason you're likely better off just styling the element tag itself or you can just use class name or id names for styling that component. The purpose of `divs` is to allow you to group elements together. Wrapping individual elements in `divs` for the purpose of styling is largely unnecessary.

shella12 commented on index.html:
```diff
@@ -1,13 +1,47 @@
+            <h3 class="h3_heading">I'm a Software Developer</h3>
+            <p class="paragraph">I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded,don’t hesitate to contact me.</p>
+            <div>
+                <ul class="social">
+                    <li class="lists"><a href="#"  > <img src="./resources/logo1.png" alt="logo"> </a></li>
+                    <li class="lists"><a href="#"  > <img src="./resources/logo2.png" alt="logo"> </a></li>
+                    <li class="lists"><a href="#" > <img src="./resources/logo3.png" alt="logo"> </a></li>
+                    <li class="lists"><a href="#"  > <img src="./resources/logo4.png" alt="logo"> </a>></li>
+                    <li class="lists"><a href="#"  > <img src="./resources/logo5.png" alt="logo"> </a></li>
+                </ul>
+            </div>
```
- [ ] [OPTIONAL] I would suggest you to remove `div` tag from around another container tag i.e. `ul` .  Both divs and unordered lists are block-level elements.  There really is no need whatsoever for the `<ul> `to be wrapped in a `div`; anything you can do with that `div` you can do with the `<ul>`. The purpose of `divs` is to allow you to group elements together and the purpose of` ul` is to group` li` elements together. Wrapping individual elements in `divs` for the purpose of styling is largely unnecessary.

Shahid-Bagwan replied:
@shella12 thankyou!

shella12 reviewed (approved):
Hi @Shahid-Bagwan, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media0.giphy.com/media/3kuSo744UIPJjcJUEn/giphy.gif"/>

## Highlights

- Great design implementation 👍
- Clean code  ✨ 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
